code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -42,6 +42,28 @@
 						else
 							H.easy_randmut(POSITIVE)
 						H.domutcheck()
+				if(HAS_TRAIT(H, TRAIT_IPCRADBRAINDAMAGE)) // instead, give IPCs brain damage
```

Comment:
I don't see why you'd want to use traits for this, you should just be able to use `isipc(L)`